[2025-09-24T09:11:29.231+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_c6 manual__2025-09-24T09:09:29.387528+00:00 [queued]>
[2025-09-24T09:11:29.255+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_c6 manual__2025-09-24T09:09:29.387528+00:00 [queued]>
[2025-09-24T09:11:29.256+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-09-24T09:11:29.281+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanse_data_c6> on 2025-09-24 09:09:29.387528+00:00
[2025-09-24T09:11:29.293+0000] {standard_task_runner.py:60} INFO - Started process 2618 to run task
[2025-09-24T09:11:29.302+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'cleanse_data_c6', 'manual__2025-09-24T09:09:29.387528+00:00', '--job-id', '561', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmp0_151e_o']
[2025-09-24T09:11:29.304+0000] {standard_task_runner.py:88} INFO - Job 561: Subtask cleanse_data_c6
[2025-09-24T09:11:29.397+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_c6 manual__2025-09-24T09:09:29.387528+00:00 [running]> on host feaba2204362
[2025-09-24T09:11:29.484+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='cleanse_data_c6' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T09:09:29.387528+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-24T09:09:29.387528+00:00'
[2025-09-24T09:11:29.674+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:11:29.688+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=cleanse_data_c6, execution_date=20250924T090929, start_date=20250924T091129, end_date=20250924T091129
[2025-09-24T09:11:29.720+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:11:29.768+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
